azure-spring-apps-maven-plugin/src/main/java/com/microsoft/azure/maven/springcloud/config/AppDeploymentRawConfig.java

```diff
@@ -19,4 +22,13 @@ public class AppDeploymentRawConfig {
     private String deploymentName;
     private String jvmOptions;
     private String runtimeVersion;
+
+    public void saveSpringCloudDeployment(@Nonnull final SpringCloudDeployment activeDeployment) {
```

Comment:
rename to `from(...)`? see also `SpringCloudAppConfig/SpringCloudDeploymentConfig.from(xxx)`

Reply:
-> merge